README.md

```diff
@@ -1,15 +1,18 @@
-# Documentation Search
+# docsearch.js
 
-Add a search autocomplete to your documentation.
+Add a search autocomplete to any documentation.
+
+Currently on-demand, send an email to [docsearch.js@algolia.com](mailto:docsearch.js@algolia.com)
```

Comment:
Maybe just docsearch@, not docsearch.js@. The js part is just the front-end, but could not work without the backend crawler.